sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/DefaultSchedulerTest.java

```diff
@@ -74,6 +77,8 @@
     private ArgumentCaptor<Collection<Protos.Offer.Operation>> operationsCaptor2;
     public static final SchedulerFlags flags = OfferRequirementTestUtils.getTestSchedulerFlags();
 
+    private static final String TASK_IP = "0.0.0.0";
+
```

Comment:
Nit: Not really re-doing CI for this, but `0.0.0.0` actually means something in the context of CNI. Someone looking at the code may thing there's an implicit meaning here.